Library/Homebrew/test/emoji_spec.rb

```diff
@@ -4,8 +4,15 @@
   describe "#install_badge" do
     subject { described_class.install_badge }
 
-    it "returns 🍺 by default" do
-      expect(subject).to eq "🍺"
+    context "with emoji environment variables unset" do
```

Comment:
I think you can skip the `context` here but CC @reitermarkus @alyssais for RSpec thoughts. Maybe worth adding another test to verify that `HOMEBREW_NO_EMOJI` and `HOMEBREW_INSTALL_BADGE` work as expected too? Feel free to punt on that though.